client/src/Core/Components/MainLayout.tsx

```diff
@@ -53,7 +54,15 @@ const MainLayoutInner = ({ styles }: IStylesOnly<MainLayoutStyles>): JSX.Element
         <div className={classes.content}>
           {usersStore.userDetails.role === 'teacher' && !asStudent ? (
             <>
-              <Navbar />
+              <Text variant="xLargePlus" className={classes.assignmentTitle}>
+                {assignmentStore.assignment.name}
+              </Text>
+              <div className={classes.navAndControlArea}>
+                <NavBarBase.NavbarTop />                  
+                <PublishControlArea/>
+              </div>
+              <Separator className={classes.separator} />

```

Comment:
what is the purpose of this Separator?

Reply:
With the new designs, there was a horizontal line separating the navigation + control area from the main layout. The `Separator` element serves the same purpose. Visually, it is like the Separator element in the `MicrosoftLearnPage.tsx` right below the selected items list.

Reply:
Just noticed that the `stickeyheader` style was achieving a similar thing. Now using the latter in my changes as well!